changelogs/unreleased/fix-escaping-issue-in-install-docs.yml

```
@@ -0,0 +1,5 @@
+description: Fix dollar sign escaping issue in installation documentation
+change-type: patch
+destination-branches: [master]
```

```suggestion
destination-branches: [master, iso4]
```
Or am I missing something?

---

This is the installation documentation for inmanta OSS, so `master` should be sufficient.

---

I see it's present on the iso4 branch as well but I assume it gets replaced with the iso instructions then?

---

Yes, it is.